webpack/config.dev.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const ReactRefreshPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
 const ESLintPlugin = require('eslint-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```

Comment:
Ideally, we would like to isolate changes in each pull request to relate as much as possible to the issue. It seems like the changes in this file aren't related to the Editor component refactor, so I might remove these!